src/main/java/net/glowstone/GlowWorld.java

```diff
@@ -1184,8 +1186,52 @@ public double getHumidity(int x, int z) {
     public <T extends Entity> T spawn(Location location, Class<T> clazz) throws IllegalArgumentException {
         GlowEntity entity = null;
 
-        if (TNTPrimed.class.isAssignableFrom(clazz)) {
+        if (LivingEntity.class.isAssignableFrom(clazz)) {
```

Comment:
You can even refer to the [old PR that I had for the original Glowstone project](https://github.com/GlowstoneMC/Glowstone/pull/282/files#diff-cea1d25e5518f2495b11e2d20021b0a0R1035).